docs/integrations/saas-cloud/asana.md

```diff
@@ -9,7 +9,7 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
 
 <img src={useBaseUrl('img/send-data/asana-icon.png')} alt="asana-icon" width="40" />
 
-The Sumo Logic app for Asana enables you to monitor and analyze the Asana data to gain insights into user and team management updates, track user log-in activities, detect data access and export, and changes to security settings and user roles, which in turn helps you to detect potential security threats. This app provides a pre-built dashboard that allows users to visualize their Asana data in a meaningful way and identify trends and patterns in the team. By taking appropriate actions based on these insights, you can ensure the security of Asana data and prevent data breaches.
+The Sumo Logic app for Asana enables you to track user log-in activities, detect data access and export, and identify changes to security settings and user roles. This allows you to gain insights into user and team management updates and identify potential security threats. This app provides a pre-built dashboard that allows you to visualize the Asana data in a meaningful way and identify trends and patterns in the team. By taking appropriate actions based on these insights, you can ensure the security of Asana data and prevent data breaches.
```

Comment:
```suggestion
The Sumo Logic app for Asana enables you to track user sign in activities, detect data access and export, and identify changes to security settings and user roles. This allows you to gain insights into user and team management updates and identify potential security threats. This app provides a pre-built dashboard that allows you to visualize the Asana data in a meaningful way and identify trends and patterns in the team. By taking appropriate actions based on these insights, you can ensure the security of Asana data and prevent data breaches.
```

Reply:
We should avoid using "log in, login, log into, log on, and so on..unless these terms appear in the UI (and you're writing instructions). Instead, we can use "sign in" here as a verb and not as a noun or adjective (sign-in, sign-out).